subtrack.MAUI/Services/Abstractions/IMonthlyPageCalculator.cs

```diff
@@ -0,0 +1,10 @@
+﻿using subtrack.DAL.Entities;
+using subtrack.MAUI.Responses;
+
+namespace subtrack.MAUI.Services.Abstractions
+{
+    public interface IMonthlyPageCalculator
+    {
+        Task<IEnumerable<IGrouping<int, SubscriptionsMonthResponse>>> GetMonthlySubscriptionLists(DateTime fromIncludedDate, DateTime toIncludedDate);
```

Comment:
This is a weird type to deal with. The submonthitem already contains the year and the items are not only grouped by year, they are grouped by SubMonth but we havent introduced a type that represents that. Would use a dict in that case